src/cm1.F

```diff
@@ -4381,10 +4381,19 @@ program cm1
         sum = 0.0
         call MPI_REDUCE(time_parceli ,sum,1,MPI_REAL,MPI_SUM,0,MPI_COMM_WORLD,ierr)
         time_parceli = sum/float(numprocs)
+
         sum = 0.0
         call MPI_REDUCE(time_droplet ,sum,1,MPI_REAL,MPI_SUM,0,MPI_COMM_WORLD,ierr)
         time_droplet = sum/float(numprocs)
```

Comment:
Does `time_droplet` include the timing of `time_lb` and `time_droplet_inject` below?

Reply:
No, the time_droplet is independent of either time_lb (Time spent in load imbalance) or the injection time.  I wanted to see how expensive the droplet injection calculation really was.  We can certainly remove the separate time_drolet_inject once we are done optimizing the entire Droplet calculation.

Reply:
Thanks for the clarification. Clear to me now.